.deploypr.yml

```diff
@@ -0,0 +1,13 @@
+---
+ deploy_on:
+   - comment
+ build:
+   commands:
+     - make
```

Comment:
make have no target

Reply:
"make build-base" (build base image)
"make build-env" (run base container)
with these two targets do i have to add this also ? 
"make up" (docker-compose up)